### PR TITLE
[PF-807] feat: add github-pr-comment-build plugin

### PR DESCRIPTION
**Pull request by renanqts**
# Motivation
https://emnify.atlassian.net/secure/RapidBoard.jspa?rapidView=10&projectKey=PF&modal=detail&selectedIssue=PF-826
https://github.com/EMnify/jenkins-shared-library/pull/72

# Description
- add github-pr-comment-build plugi

### CODE REVIEW

**Comment by StephenKing** [comment]
Any particular reason to not use this? https://github.com/jenkinsci/pipeline-github-plugin#issuecommenttrigger

**Comment by renanqts** [comment]
> Any particular reason to not use this? https://github.com/jenkinsci/pipeline-github-plugin#issuecommenttrigger

I explained in the liked PR

File: plugins.txt [hunk 1]
```diff
@@ -20,4 +20,4 @@ ssh-agent
 theme-manager
 timestamper
 workflow-aggregator
-
+github-pr-comment-build
```
Comment (IppX) [inline plugins.txt:23]: It's ordered alphabetically, please put it line 6 ;)

**Review by IppX — CHANGES_REQUESTED** [review]

File: plugins.txt (hunk 1 above)
Comment (renanqts) [reply plugins.txt:23]: hahaha... okay

**Review by IppX — APPROVED** [review]

**Pull request merged**